bridgetown-core/lib/bridgetown-core/configuration.rb

```diff
@@ -148,6 +148,8 @@ def safe_load_file(filename)
         Tomlrb.load_file(filename)
       when %r!\.ya?ml!i
         YAMLParser.load_file(filename) || {}
+      when %r!\.rb!i
+        Bridgetown.load_configure
```

Comment:
Should I name it `load_file` for consistency with the other two options?
```suggestion
        Bridgetown.load_file
```

Reply:
Maybe `load_configuration_file`?

Reply:
changed, I tried to write a test to confirm Bridgetown is reading the `bridgetown.config.rb` file but I wasn't able to get it working, would you give me a  clue what should I do, please?